src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx

```diff
@@ -39,6 +43,11 @@ const BaseConfiguration = () => {
   const [toastMessage, setToastMessage] = useState<string>("");
   const [toastType, setToastType] = useState<string>("danger");
   const { updateRouteHandlerState } = routeHandlerStore;
+  const [currentScoring, setCurrentScoring] = useState<string>();
+  const [changeScoringDialog, setChangeScoringDialog] =
+    useState<ChangeScoringDialog>({
+      open: false,
```

Comment:
Any reason we need a ChangeScoringDialog instead of directly using a boolean value ?

Reply:
@RohitKandimalla Just in case we need any other variations, e.g. messages based on scoring, etc.

Reply:
@RohitKandimalla Just changed it to boolean. Thanks Rohit!